cluster/images/hyperkube/Dockerfile

```diff
@@ -40,28 +38,22 @@ RUN cp /usr/bin/nsenter /nsenter
 COPY hyperkube /hyperkube
 
 # Manifests for the docker guide
-COPY static-pods/master.json /etc/kubernetes/manifests/
-COPY static-pods/etcd.json /etc/kubernetes/manifests/
-COPY static-pods/addon-manager.json /etc/kubernetes/manifests/
-
-# TODO: Move out kube-proxy to a DaemonSet again
-COPY static-pods/kube-proxy.json /etc/kubernetes/manifests/
+COPY static-pods/master.json \
+     static-pods/etcd.json \
+     static-pods/addon-manager-singlenode.json \
+     static-pods/kube-proxy.json \
+        /etc/kubernetes/manifests/
```

Comment:
This is some limitation of Dockerfile engine because copying multiple files from subdirectory does not work.
```
Step 5 : COPY static-pods/master.json static-pods/etcd.json static-pods/addon-manager-singlenode.json static-pods/kube-proxy.json /etc/kubernetes/manifests/
lstat static-pods/master.json: no such file or directory
Makefile:56: recipe for target 'build' failed
```
Does it work on your side ?